packages/playground/Samples/accessible.tsx

```diff
@@ -16,41 +25,70 @@ export default class Bootstrap extends React.Component<
     this.state = {displayText: 'Starting text. (THIRD ITEM)'};
   }
 
+  myElement = React.createRef();
+
   render() {
     return (
       <View style={styles.container}>
         <View
           style={styles.item}
+          accessibilityLabel="FIRST ITEM"
           {...{
             // Use weird format as work around for the fact that these props are not part of the @types/react-native yet
             focusable: true,
-            accessibilityLabel: 'FIRST ITEM',
           }}>
           <Text style={styles.text}>Welcome to React Native! (FIRST ITEM)</Text>
         </View>
         <View
           style={styles.item}
+          accessibilityLabel="SECOND ITEM"
           {...{
             // Use weird format as work around for the fact that these props are not part of the @types/react-native yet
             enableFocusRing: false,
             focusable: true,
-            accessibilityLabel: 'SECOND ITEM',
           }}>
           <Text style={styles.text}>No focus visual (SECOND ITEM)</Text>
         </View>
         <View
           style={styles.item}
+          accessibilityLabel="THIRD ITEM"
           {...{
             // Use weird format as work around for the fact that these props are not part of the @types/react-native yet
             onFocus: () => this.setState({displayText: 'FOCUSED'}),
             onBlur: () => this.setState({displayText: 'BLURRED'}),
             enableFocusRing: false,
             focusable: true,
-            accessibilityLabel: 'THIRD ITEM',
           }}>
           <Text style={styles.text}>{this.state.displayText}</Text>
         </View>
-        <TextInput />
+        <TouchableHighlight
+          style={styles.item}
+          accessibilityLabel="TEST Announce For Accessibility"
+          onPress={() => {
+            AccessibilityInfo.announceForAccessibility('Testing Testing 1 2 3');
```

Comment:
As an FYI, we already have test UI to do this. https://github.com/microsoft/react-native-windows/blob/003602b90aeda1b2a43702c6a9307c0f01010e59/packages/%40react-native-windows/tester/src/js/examples-win/Accessibility/AccessibilityInfoExample.js#L37